packages/components/src/components/Tooltip/index.tsx

```diff
@@ -1,97 +1,156 @@
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 import React, { ReactNode } from 'react';
-import Tippy, { TippyProps } from '@tippy.js/react';
-import { tippy } from './tippy.style';
-import colors from '../../config/colors';
+import { useSpring, animated } from 'react-spring';
+import Tippy, { TippyProps } from '@tippyjs/react/headless';
+import { Instance, Props as TProps } from 'tippy.js';
 import { Link } from '../typography/Link';
 import { FONT_SIZE, FONT_WEIGHT } from '../../config/variables';
-import { SuiteThemeColors } from '../../support/types';
 
-const tooltipGlobalStyles = css`
-    ${tippy}
+type Cursor = 'inherit' | 'pointer' | 'help' | 'default';
 
-    .tippy-tooltip {
-        background: ${props => props.theme.BG_TOOLTIP};
-        color: ${props => props.theme.TYPE_WHITE};
-        font-weight: ${FONT_WEIGHT.MEDIUM};
-        border-radius: 5px;
-        font-size: ${FONT_SIZE.TINY};
-        text-align: left;
-        box-shadow: 0 3px 14px 0 rgba(0, 0, 0, 0.15);
+const Wrapper = styled.div``;
 
-        .tippy-arrow {
-            border: 5px solid transparent;
-        }
-    }
+const TooltipBoxDefault = styled(animated.div)<{ maxWidth: number }>`
+    padding: 4px 5px;
+    background: ${props => props.theme.BG_TOOLTIP};
+    color: ${props => props.theme.TYPE_WHITE};
+    font-weight: ${FONT_WEIGHT.MEDIUM};
+    border-radius: 5px;
+    font-size: ${FONT_SIZE.TINY};
+    text-align: left;
+    box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2);
+    max-width: ${props => props.maxWidth}px;
+`;
 
-    .tippy-tooltip[data-placement^='top'] > .tippy-arrow {
-        border-top-color: ${props => props.theme.BG_TOOLTIP};
-    }
+const BoxDefault = (props: React.ComponentProps<typeof TooltipBoxDefault>) => (
+    <TooltipBoxDefault {...props} />
+);
 
-    .tippy-tooltip[data-placement^='bottom'] > .tippy-arrow {
-        border-bottom-color: ${props => props.theme.BG_TOOLTIP};
-    }
+const TooltipBoxRich = styled(animated.div)<{ maxWidth: number }>`
+    padding: 24px;
+    background: ${props => props.theme.BG_WHITE_ALT};
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    border-radius: 5px;
+    font-size: ${FONT_SIZE.NORMAL};
+    text-align: left;
+    box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.2);
+    max-width: ${props => props.maxWidth}px;
+`;
 
-    .tippy-tooltip[data-placement^='left'] > .tippy-arrow {
-        border-left-color: ${props => props.theme.BG_TOOLTIP};
-    }
+const BoxRich = (props: React.ComponentProps<typeof TooltipBoxRich>) => (
```

Comment:
Couldn't we directly use `TooltipBoxRich` component as this component doesn't really add anything?